docs/specs/tsk-ha5iau/2026-08-28-lora-off-grid-transport-design.md

```
@@ -0,0 +1,226 @@
+  "metadata": {
+    "device_id": "<hex-mac-or-uuid>",
+    "sequence": <uint32>,
+    "timestamp": <unix-timestamp>,
+    "hop_limit": <uint8>,
+    "channel": "<meshtastic-channel-name>"
+  }
+}
+```
+
+### Size Analysis
+| Component | Size (bytes) | Notes |
```

markdownlint-cli2 (0.23.2)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)